package.json

```diff
@@ -178,7 +178,7 @@
     "cozy-pouch-link": "16.12.1",
     "cozy-realtime": "3.11.0",
     "cozy-scripts": "5.2.0",
-    "cozy-stack-client": "16.12.0",
+    "cozy-stack-client": "16.19.0",
```

Comment:
You do not need to manually manage the versions, semantic-release automatically detects which kind of release it needs to make (based on commits "feat" / "fix" etc...). Please remove those lines :)